lua/telescope/builtin/files.lua

```diff
@@ -339,17 +345,22 @@ files.file_browser = function(opts)
 
   pickers.new(opts, {
     prompt_title = "File Browser",
-    finder = opts.new_finder(opts.cwd),
+    finder = opts.new_finder { path = opts.cwd },
     previewer = conf.file_previewer(opts),
     sorter = conf.file_sorter(opts),
     attach_mappings = function(prompt_bufnr, map)
       action_set.select:replace_if(function()
         return is_dir(action_state.get_selected_entry().path)
       end, function()
-        local new_cwd = vim.fn.expand(action_state.get_selected_entry().path:sub(1, -2))
+        local new_cwd = vim.loop.fs_realpath(action_state.get_selected_entry().path)
```

Comment:
I think i proposed this fix because i thought its a good idea to normalize the path with it but i actually think that might be wrong. `plenary.path:normalize` should exist now.

I am not sure. how we should handle symbolic links if i am in `/home/conni/a` and open file `link` which goes to `/home/conni/b` then i expect `../` to take me back to `/home/conni/a` right and not to `/home/conni`